torch/csrc/api/include/torch/nn/modules/dropout.h

```diff
@@ -64,6 +64,19 @@ class TORCH_API FeatureDropoutImpl
   void pretty_print(std::ostream& stream) const override;
 };
 
+class TORCH_API AlphaDropoutImpl
+    : public detail::DropoutImplBase<AlphaDropoutImpl> {
+ public:
+  explicit AlphaDropoutImpl(const DropoutOptions& options_ = DropoutOptions());
```

Comment:
@Suyash458 Thanks so much for the awesome work! I think we can change the arguments in `DropoutOptions` to include `p` and `inplace` instead of `rate` to match the Python version better. And we can have `using AlphaDropoutOptions = DropoutOptions;` and then use `AlphaDropoutOptions` everywhere in the AlphaDropout code :D

Reply:
@yf225 Thanks for having a look! Adding `p` and `inplace` makes sense, I'll add the extra params to `DropoutOptions`. This'll affect `DropoutImpl` and `FeatureDropoutImpl` as well

Reply:
Yes and I think we should rename `rate` to `p` in `DropoutOptions`, and it is okay to break backward compatibility in favor of code simplicity and parity with Python API